Documentation/Index.rst

```diff
@@ -1,76 +1,70 @@
-.. -*- coding: utf-8 -*- with BOM.
-.. include:: Includes.txt
+.. include:: /Includes.rst.txt
 
-.. _start:
+====
+Surf
+====
 
-TYPO3 Surf |version| documentation
-==================================
+:Package name:
```

Comment:
Surf is no package. It's an application

Reply:
Yes, but this naming convention refers to **Composer** package name.